src/components/application_manager/src/commands/command_request_impl.cc

```diff
@@ -194,6 +194,41 @@ bool CommandRequestImpl::CheckSyntax(const std::string& str,
   return true;
 }
 
+smart_objects::SmartObject CreateUnsupportedResourceResponse(
+    const hmi_apis::FunctionID::eType function_id,
+    const uint32_t hmi_correlation_id) {
+  smart_objects::SmartObject response;
+  response[strings::params][strings::message_type] = MessageType::kResponse;
+  response[strings::params][strings::correlation_id] = hmi_correlation_id;
+  response[strings::params][strings::protocol_type] =
+      CommandImpl::hmi_protocol_type_;
+  response[strings::params][strings::protocol_version] =
+      CommandImpl::protocol_version_;
+  response[strings::params][strings::function_id] = function_id;
+  response[strings::params][hmi_response::code] =
+      hmi_apis::Common_Result::UNSUPPORTED_RESOURCE;
+  return response;
+}
+
+bool CommandRequestImpl::ProcessHMIInterfacesAvailability(
+    const uint32_t hmi_correlation_id,
+    const hmi_apis::FunctionID::eType& function_id) {
+  HmiInterfaces& hmi_interfaces = application_manager_.hmi_interfaces();
+  HmiInterfaces::InterfaceName interface =
+      hmi_interfaces.GetInterfaceFromFunction(function_id);
+  DCHECK(interface != HmiInterfaces::HMI_INTERFACE_INVALID_ENUM);
```

Comment:
@pestOO I think in case if this method does not know any about this function in release mode it is better to process this function but not say that interface of this function is not available